src/Piral.Blazor.Tools/content/src/mocks/debug.js

```diff
@@ -0,0 +1,67 @@
+const { resolve, basename } = require('path');
+const { proxyRequest } = require('kras/utils');
+
+const configuration = '**MSBUILD_Configuration**';
+const targetFramework = '**MSBUILD_TargetFramework**';
+
+const bbjson = 'blazor.boot.json';
+const pjson = 'package.json';
+const lsjson = 'launchSettings.json';
+
+const piralPiletFolder = resolve('.');
+const rootFolder = resolve(piralPiletFolder, '..', '..');
+const blazorfolderName = basename(piralPiletFolder);
+const blazorprojectfolder = resolve(rootFolder, blazorfolderName);
+const bbjsonPath = resolve(blazorprojectfolder, 'bin', configuration, targetFramework, 'wwwroot', '_framework', bbjson);
+
+const appshellname = require(resolve(piralPiletFolder, pjson)).piral.name;
+const appshellAppDir = resolve(require.resolve(appshellname), '..');
+
+const originalManifest = require(resolve(appshellAppDir, '_framework', bbjson));
+const piletManifest = require(bbjsonPath);
+
+const launchSettingsPath = resolve(blazorprojectfolder, 'Properties', lsjson);
+const iisUrl = require(launchSettingsPath)?.iisSettings?.iisExpress?.applicationUrl;
```

Comment:
This wouldn't work in older Node versions (e.g., Node 10 and Node 12). Please write JS that works with Node 10 (or at least 12).